test/powershell/Modules/Microsoft.PowerShell.Utility/assets/TestImportCsv2.csv

```diff
@@ -1,5 +1,5 @@
-Column1,Column2,Column 3
-data1,1,A
-data2,2,B
-data3,3,C
-data4,4,D
+﻿Column1,Column2,Column 3
```

Comment:
@mirichmo  Could you explain your idea?
Now there are three files with test data:
1. TestImportCsv1.csv - w/o header.
2. TestImportCsv2.csv  - the same data with header.
3. TestImportCsv3.csv  - the same data with header and W3C header.
_Each test_ is performed on each file.